examples/unix/c11/z_get.c

```diff
@@ -12,16 +12,30 @@
 //   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
 
 #include <ctype.h>
+#include <errno.h>
+#include <pthread.h>
 #include <stddef.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
 #include <zenoh-pico.h>
 
 #if Z_FEATURE_QUERY == 1
+#define QUERY_TIMEOUT 10  // query timeout in seconds
+#define handle_error_en(en, msg) \
+    do {                         \
+        errno = en;              \
+        perror(msg);             \
+        exit(EXIT_FAILURE);      \
+    } while (0)
+pthread_cond_t cond;
+pthread_mutex_t mutex;
+
 void reply_dropper(void *ctx) {
     (void)(ctx);
     printf(">> Received query final notification\n");
+    pthread_cond_signal(&cond);
+    pthread_cond_destroy(&cond);
```

Comment:
This should use the public sync mechanisms found in `include/zenoh-pico/system/platform.h`, this impacts all the following pthread_cond and pthread_mutex calls